packages/bigtable/src/table.js

```diff
@@ -425,7 +425,8 @@ Table.prototype.getMetadata = function(options, callback) {
 };
 
 /**
- * Get Row objects for the rows currently in your table.
+ * Get {module:bigtable/row} objects for the rows currently in your table as a
+ * readable object stream.
```

Comment:
I think we should link to bigtable.getRows() and remove the specific options and examples here beyond stream related. Move the examples to `getRows()` if they're not there already.